omniNotes/src/androidTest/java/it/feio/android/omninotes/helpers/Methods.java

```diff
@@ -1,12 +1,8 @@
-package it.feio.android.omninotes.Helpers;
+package it.feio.android.omninotes.helpers;
```

Comment:
This class should have only the Override methods. And a suggestive name is: EspressoMethods.
Eg: getListViewChildCount().